app/elements/candidates/invite-candidates.html

```diff
@@ -51,25 +55,21 @@
 					type: Array,
 					notify: true
 				},
-				challenge: {
-					type: Object,
-					notify: true
-				},
 				candidates: {
 					type: Array,
 					notify: true,
 					value: []
 				}
 			},
-			toggle: function(){
+			toggle: function () {
 				this._clearDialog();
 				this._addCandidate();
 				this.$.dialog.toggle();
 			},
-			inviteCandidates: function(){
-				for (var i = 0; i < this.candidates.length; i++){
+			inviteCandidates: function () {
+				for (var i = 0; i < this.candidates.length; i++) {
 					var candidate = this.candidates[i];
-					if (candidate.Removed !== undefined && candidate.Address !== '' && candidate.Challenge !== undefined){
+					if (candidate.Removed === undefined && candidate.Address !== '' && candidate.Challenge !== undefined) {
```

Comment:
No, because candidate.Removed will only ever either be undefined or true. You cannot un-remove something.